app/auth/routes.py

```
@@ -130,6 +137,203 @@ async def verify_token(current_user: Dict[str, Any] = Depends(get_current_user))
+        user_id=current_user["uid"],
+        action="session_revoked",
+        request=request,
+        details={"session_id": session_id}
+    )
+    
+    return {"message": f"Session {session_id} has been revoked"}
+
+
+@router.get("/audit-logs", response_model=List[AuditLogEntry])
+async def get_audit_logs(
+    current_user: Dict[str, Any] = Depends(get_current_user),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)